src/applications/mhv/secure-messaging/components/AttachmentsList.jsx

```diff
@@ -183,7 +183,7 @@ const AttachmentsList = props => {
                       alt="Attachment icon"
                       aria-hidden="true"
                     />
-                    <span>{file.name} </span>(
+                    <span className="attachment-name-text">{file.name}</span>(
```

Comment:
I would suggest not to include these old dir changes whatsoever. They may break cypress tests (as tests are not updated in the old dir) and cause more confusion